image-download

```diff
@@ -307,20 +323,16 @@ def download_images(image_list, force, quiet, state, store):
 
     for image in image_list:
         image = get_test_image(image)
-        try:
-            if state:
-                target = state_target(image)
-            else:
-                target = committed_target(image)
+        if state:
+            target = state_target(image)
+        else:
+            target = committed_target(image)
 
-            # don't download the same thing multiple times in parallel
-            wait_lock(target)
+        # don't download the same thing multiple times in parallel
+        wait_lock(target)
 
-            if force or state or not os.path.exists(target):
-                download(target, force, state, quiet, store)
-        except Exception as ex:
-            success = False
-            sys.stderr.write("image-download: {0}\n".format(str(ex)))
```

Comment:
Admittedly I don't quite like these bare exception handlers. But this changes behaviour, and thus it should at least be in a separate commit and justified carefully. Possibly it's just a leftover from intermediate debugging where you did want to see the full tracebacks?

Reply:
This was unintentional yes.   I'll remove it.

But I also don't like it.  Separate issue, though :)